### ISSUE TITLE
libconfig.so.9: cannot open shared object file

### ISSUE THREAD

**Issue by svoop** [glidernet/ogn-rf#39]
I'm maintaining a Gentoo Ebuild of ogn-rf. (Ebuilds are the equivalent e.g. to packages on Debian, but are compiled from source on install.)

After upping libconfig from 1.5 to 1.7.2, I get the following on `telnet localhost 50000`:

```
/usr/bin/ogn-rf: error while loading shared libraries: libconfig.so.9: cannot open shared object file: No such file or directory
```

Rebuilding `ogn-rf` does not help. In fact, there's no `libconfg.so9`, but the following:

```
/usr/lib/libconfig.so -> libconfig.so.11.0.2
/usr/lib/libconfig.so.11 -> libconfig.so.11.0.2
/usr/lib/libconfig.so.11.0.2
```

In the README, I don't see any specific version of libconfig being required. However, I don't develop in C, so I can't really tell whether there is such a constraint in the source.

Thanks a lot for your help!

**Comment by svoop** [comment]
I've downgraded libconfig back to 1.5 – works for me, but...

Gentoo has ditched this old release last December, it's therefore not possible to downgrade unless you pull the old ebuild from their repo and manually trick the download (because the old download link is no longer valid).

In other words: The ebuild is broken and cannot be installed by the average user for now.

Any chance you could make the code compatible with libconfig 1.7 (and cut a release)?

**Comment by svoop** [comment]
Appears to be fixed with libconfig-1.7.3.
